server/src/main/java/org/joda/time/format/StrictISODateTimeFormat.java

```diff
@@ -90,7 +90,7 @@ protected StrictISODateTimeFormat() {
      * <p>
      * This method examines the fields provided and returns an ISO-style
      * formatter that best fits. This can be useful for outputting
-     * less-common ISO styles, such as YearMonth (YYYY-MM) or MonthDay (--MM-DD).
+     * less-common ISO styles, such as YearMonth (yyyy-MM) or MonthDay (--MM-DD).
```

Comment:
this is the joda time class, which means this needs to remain original upper cased